src/main.rs

```diff
@@ -50,6 +50,7 @@ enum Commands {
     ///  Prints the shell function used to execute starship
     Init {
         shell: String,
+        /// Prints out the main initialization script
```

Comment:
I feel like this by itself isn't a great explanation of what the options do. The explanation should mention how this is different from not passing the option, it may not be clear to users what the main script is without context. Another option would be to hide this option, but I think I prefer keeping it visible.

Reply:
What about adding a redirect to [the page which introduces it](https://github.com/starship/starship/pull/168)? I think the option without explanation might make the user relatively more confused.